File: FullStackDevExercise/ClientApp/src/app/models/pets.ts
export class petsModel{
    id: number
    owner_id: number
    type: string 
    name: string
    age : number
    constructor()
    {
        this.id=0;
        this.owner_id=0;
        this.type=''
        this.name=''
        this.age=0
    }
}
export class petsListModel  extends petsModel
{
    owner_name :string
    constructor()
    {
        super();
        this.owner_name = '';
    }
}

File: FullStackDevExercise/models/Pets.cs
using Microsoft.Data.Sqlite;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace FullStackDevExercise.models
{
  public class PetsList : Pets
  {
    public string owner_name { get; set; }
  }
  public class Pets
  {
    public int id { get; set; }
    public int owner_id { get; set; }
    [MaxLength(50)]
    public string type { get; set; }
    [MaxLength(50)]
    public string name { get; set; }
    public int age { get; set; }

  }
}
